client/src/main/scala/drt/client/components/mui/MuiInputLabelProps.scala

```diff
@@ -0,0 +1,25 @@
+package drt.client.components.mui
+
+import scala.scalajs.js
+import scala.scalajs.js.JSConverters.JSRichGenMap
+
+@js.native
+trait MuiInputLabelProps extends js.Object {
+  def shrink: js.UndefOr[Boolean] = js.native
+  def classes: js.UndefOr[js.Object] = js.native
+}
+
+object MuiInputLabelProps {
+  def apply(
+             shrink: js.UndefOr[Boolean] = js.undefined,
```

Comment:
usually I prefer shrink to on same line as apply

Reply:
I guess we should have a style guide, I always do this, but mostly out of habit, and I don't really care that much.